src/main/java/com/minecolonies/api/blocks/ModBlocks.java

```diff
@@ -16,94 +32,101 @@
-    public static AbstractBlockMinecoloniesDefault<? extends AbstractBlockMinecoloniesDefault<?>>                   blockScarecrow;
-    public static AbstractBlockMinecoloniesHorizontal<? extends AbstractBlockMinecoloniesHorizontal<?>>             blockPlantationField;
-    public static AbstractBlockMinecolonies<? extends AbstractBlockMinecolonies<?>>                                 blockCompostedDirt;
-    public static AbstractColonyFlagBanner<? extends AbstractColonyFlagBanner<?>>                                   blockColonyBanner;
-    public static AbstractColonyFlagBanner<? extends AbstractColonyFlagBanner<?>>                                   blockColonyWallBanner;
-    public static AbstractBlockGate                                                                                 blockIronGate;
-    public static AbstractBlockGate    blockWoodenGate;
-    public static MinecoloniesFarmland farmland;
-    public static MinecoloniesFarmland floodedFarmland;
-
-    public static MinecoloniesCropBlock blockBellPepper;
-    public static MinecoloniesCropBlock blockCabbage;
-    public static MinecoloniesCropBlock blockChickpea;
-    public static MinecoloniesCropBlock blockDurum;
-    public static MinecoloniesCropBlock blockEggplant;
-    public static MinecoloniesCropBlock blockGarlic;
-    public static MinecoloniesCropBlock blockOnion;
-    public static MinecoloniesCropBlock blockSoyBean;
-    public static MinecoloniesCropBlock blockTomato;
-    public static MinecoloniesCropBlock blockRice;
+    public static final DeferredBlock<BlockConstructionTape> blockConstructionTape = simpleBlockItem("blockconstructiontape", BlockConstructionTape::new);
+    public static final DeferredBlock<BlockMinecoloniesRack> blockRack = simpleBlockItem("blockminecoloniesrack", BlockMinecoloniesRack::new);
+    public static final DeferredBlock<BlockMinecoloniesGrave> blockGrave = simpleBlockItem("blockminecoloniesgrave", BlockMinecoloniesGrave::new);
+    public static final DeferredBlock<BlockMinecoloniesNamedGrave> blockNamedGrave = simpleBlockItem("blockminecoloniesnamedgrave", BlockMinecoloniesNamedGrave::new);
+    public static final DeferredBlock<BlockWaypoint> blockWayPoint = simpleBlockItem("blockwaypoint", BlockWaypoint::new);
+    public static final DeferredBlock<BlockBarrel> blockBarrel = simpleBlockItem("barrel_block", BlockBarrel::new);
+    public static final DeferredBlock<BlockDecorationController> blockDecorationPlaceholder = simpleBlockItem("decorationcontroller", BlockDecorationController::new);
+    public static final DeferredBlock<BlockScarecrow> blockScarecrow = simpleBlockItem("blockhutfield", BlockScarecrow::new);
+    public static final DeferredBlock<BlockPlantationField> blockPlantationField = simpleBlockItem("blockhutplantationfield", BlockPlantationField::new);
+    public static final DeferredBlock<BlockCompostedDirt> blockCompostedDirt = simpleBlockItem("composted_dirt", BlockCompostedDirt::new);
```

Comment:
the formatting here is quite painful

Reply:
hmm? that's fine one-liner for me

Reply:
one liner is fine, but I'd align them that the = are ont he same line, makes it easier to see the ids etc